### PR TITLE
[Merged by Bors] - ET-3616 Fix workflows

### PR DESCRIPTION
**Pull request by Robert-Steiner**
fix workflows for ingestion and personalization
https://xainag.atlassian.net/browse/ET-3616

### CODE REVIEW

**Comment by Robert-Steiner** [comment]
https://github.com/xaynetwork/xayn_discovery_engine/actions/runs/3481157410
https://github.com/xaynetwork/xayn_discovery_engine/actions/runs/3481204460
i also checked the docker images locally

File: web-api/Dockerfile [hunk 1]
```diff
@@ -2,7 +2,9 @@ FROM debian:bookworm-slim
 ARG api
 LABEL api="$api"
 WORKDIR /app
-#renamed as $api doesn't work in ENTRYPOINT
-COPY ./"$api" ./server.bin
+# copy everything in the cwd (bin and assets)
+COPY ./ ./
+# renamed as $api doesn't work in ENTRYPOINT
+RUN mv ./"$api" ./server.bin
```
Comment (acrrd) [inline web-api/Dockerfile:8]: :question: 
Is it worth it to have a parameter only to rename a file? When we generate the docker context we can already name the binary as server.

**Review by acrrd — APPROVED** [review]

File: web-api/Dockerfile (hunk 1 above)
Comment (Robert-Steiner) [reply web-api/Dockerfile:8]: i can remove it

Comment (Robert-Steiner) [reply web-api/Dockerfile]: done. was a good idea. it makes the pulling faster

**Comment by Robert-Steiner** [comment]
bors merge

**Pull request closed without merging**